libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html

```diff
@@ -1,72 +1,98 @@
-			<ix-select-item label="Item 1" value="1"></ix-select-item>
-			<ix-select-item label="Item 2" value="2"></ix-select-item>
-			<ix-select-item label="Item 3" value="3"></ix-select-item>
-		</ix-select>
-		<div class="invalid-feedback">Please select an asset</div>
-
-		<ix-input-group>
-			<span slot="input-start">Telephone:</span>
-			<input
-				placeholder="111-111-111"
-				type="tel"
-				name="phone"
-				[(ngModel)]="createCaseForm.phone"
-				pattern="[+]?[0-9]+"
-				maxlength="16"
+			<ix-select
+				i18nPlaceholder="Select a Facility"
+				i18nSelectListHeader="Please select a facility"
+				name="selectFacility"
+				[(ngModel)]="createCaseForm.selectFacility"
 				required
-			/>
-			<div class="valid-feedback">Looks good!</div>
-			<div class="invalid-feedback">Please provide a valid phone number</div>
-		</ix-input-group>
+			>
+				@for (facility of facilities; track facility.id) {
+					<ix-select-item
+						[label]="facility.heading"
+						[value]="facility.id"
+					></ix-select-item>
+				}
+			</ix-select>
+			<div class="invalid-feedback">Please select a facility</div>
 
-		<ix-input-group>
-			<span slot="input-start">Email:</span>
-			<input
-				placeholder="example@domain.com"
-				type="email"
-				name="email"
-				[(ngModel)]="createCaseForm.email"
+			<ix-select
+				i18nPlaceholder="Select an Asset"
+				name="selectAsset"
+				[(ngModel)]="createCaseForm.selectAsset"
 				required
-			/>
-			<div class="valid-feedback">Looks good!</div>
-			<div class="invalid-feedback">Please provide a valid email address</div>
-		</ix-input-group>
+			>
+				<!--TODO: insert assets here-->
+				<ix-select-item label="Item 1" value="1"></ix-select-item>
+				<ix-select-item label="Item 2" value="2"></ix-select-item>
+				<ix-select-item label="Item 3" value="3"></ix-select-item>
+			</ix-select>
+			<div class="invalid-feedback">Please select an asset</div>
+
+			<ix-input-group>
+				<span slot="input-start">Telephone:</span>
+				<input
+					placeholder="111-111-111"
+					type="tel"
+					name="phone"
+					[(ngModel)]="createCaseForm.phone"
+					pattern="[+]?[0-9]+"
+					maxlength="16"
+					required
+				/>
+				<div class="valid-feedback">Looks good!</div>
+				<div class="invalid-feedback">Please provide a valid phone number</div>
+			</ix-input-group>
+
+			<ix-input-group>
+				<span slot="input-start">Email:</span>
+				<input
+					placeholder="example@domain.com"
+					type="email"
+					name="email"
+					[(ngModel)]="createCaseForm.email"
+					required
+				/>
+				<div class="valid-feedback">Looks good!</div>
+				<div class="invalid-feedback">Please provide a valid email address</div>
+			</ix-input-group>
+
+			<ix-input-group>
+				<span slot="input-start">Additional Information:</span>
+				<input type="text" name="text" [(ngModel)]="createCaseForm.text" />
+			</ix-input-group>
+
+			<ix-button type="submit"> Create Case </ix-button>
+		</form>
+
+		<div *ngIf="getFacilityValue() !== undefined">
```

Comment:
Do it like this:
```
*ngIf="getFacilityValue() as facility"
```
that makes it so only if it is defined something is shown and the variable can be used later instead of all the getFacility() calls